packages/ai-jsx/src/batteries/docs.tsx

```diff
@@ -600,6 +600,45 @@ export class LocalCorpus<
   }
 }
 
+/** A fully mananged {@link Corpus} served by Fixie. */
```

Comment:
It's fine if it's a follow-up PR, but let's add a guide/tutorial handholding the reader through setting up a Fixie corpus + using it here. Maybe @mdwelsh can help with that.

Reply:
Yep, will do (though any reasonable directions depend on my next backend PR).